NYCJobsWeb/NYCJobsWeb.csproj.user

```diff
@@ -9,6 +9,7 @@
     <WebStackScaffolding_IsAsyncSelected>False</WebStackScaffolding_IsAsyncSelected>
     <WebStackScaffolding_ViewDialogWidth>600</WebStackScaffolding_ViewDialogWidth>
     <NameOfLastUsedPublishProfile>NYCJobsWeb - Web Deploy</NameOfLastUsedPublishProfile>
+    <UseIISExpress>true</UseIISExpress>
```

Comment:
@liamca You should not add `.user` files to the repo. Please add the `.user` file extension to `.gitignore`.